examples/chef/devices/rootnode_contactsensor_27f76aeaf5.matter

```diff
@@ -1407,6 +1407,491 @@ cluster BooleanState = 69 {
   readonly attribute int16u clusterRevision = 65533;
 }
 
+/** The Test Cluster is meant to validate the generated code */
+internal cluster UnitTesting = 4294048773 {
```

Comment:
I thought chef was supposed to be "minimal example".

How about creating a separate chef example that sepecifically has only unit test (or unit test beside some standard thing ... however at least it is a dedicated sample and not overloading contactsensor)

Reply:
This was done in collaboration w/ user of the sample.